static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.ts

```diff
@@ -93,7 +91,7 @@ export const init = (): Promise<void> => {
 			},
 		);
 
-		onConsentChange((state) => {
+		void getInitialConsentState().then((state) => {
```

Comment:
I think you’ll want to simply `return` this chain of promises, here?

Reply:
In a subsequent PR we'll want to return this promise but for now I think the return values of `init` and `setupAdvertising` shouldn't change.